.agents/skills/use-railway/scripts/pg-extensions.py

```
@@ -0,0 +1,370 @@
+
+    return extensions
+
+
+def get_extension_dependencies(service: str, extension: str) -> List[str]:
+    """Get dependencies for an extension."""
+    query = f"""
+        SELECT DISTINCT unnest(pev.requires) as dependency
+        FROM pg_available_extension_versions pev
+        WHERE pev.name = '{extension}' AND pev.requires IS NOT NULL
+        ORDER BY dependency
+    """
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)